torchaudio/prototype/tacotron2.py

```diff
@@ -947,3 +1070,35 @@ def forward(
             gate_outputs.masked_fill_(mask[:, 0, :], 1e3)
 
         return mel_specgram, mel_specgram_postnet, gate_outputs, alignments
+
+    @torch.jit.export
+    def infer(self, text: Tensor, text_lengths: Tensor) -> Tuple[Tensor, Tensor, Tensor]:
+        r"""Using Tacotron2 for inference. This is generally used for generating mel spectrograms.
```

Comment:
Can you describe what this function does and/or how this is expected to be used?

Reply:
Thanks for the suggestion, it's added [here](https://github.com/pytorch/audio/pull/1648/commits/e2315340dee97e812ce5cd066c0c6de119608cd0).